js/components/Table/table.scss

```diff
@@ -2,8 +2,7 @@
  * Table
  */
 @import '~@dosomething/forge/scss/toolkit';
-@import './sass/app';
-
+@import '../../../sass/app';
```

Comment:
here it is importing from `app.scss` which then imports the fonts so I'm not sure why the error says it's trying to get the fonts directly from this file? 

```
ERROR in ./js/components/Table/table.scss
Module build failed: ModuleNotFoundError: Module not found: Error: Can't resolve '../site/fonts/leaguegothic-regular.woff' in '/Users/chloelee/DoSomething.org/domino/js/components/Table'
``` 

The error makes sense since it doesn't have the correct path but that is relative to `app.scss` which it is importing?